multiple-sites/.env.deployer

```diff
@@ -19,5 +19,9 @@ AUTH_SALT='generateme'
 SECURE_AUTH_SALT='generateme'
 LOGGED_IN_SALT='generateme'
 NONCE_SALT='generateme'
+
+# Setting variable when same redis server is used for discreet WP installs 
```

Comment:
@mbtamuli This is my concern!

By design if we are giving separate MySQL to each one then maybe its better to give separate Redis also. That means key can generic like `wp:` or `wordpress:`